File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> hash = new HashMap<>();
        int[] arr = {0,0};
        for(int i = 0;i<nums.length;i++){
            if(hash.containsKey(target-nums[i])){
              arr[0] = hash.get(target-nums[i]);
              arr[1] = i;
              break;
            }
            hash.put(nums[i],i);
        }
        return arr;
    }
}

File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for a in range(len(nums)-1):
          for b in range(a,len(nums)-1):
            if((nums[a] + nums[b+1]) == target):
             self = [a,b+1]
             return self
 

         